platform/jvm/common/src/main/kotlin/io/bitdrift/capture/common/Runtime.kt

```diff
@@ -69,6 +69,43 @@ sealed class RuntimeFeature(
     data object DROPPED_EVENTS_MONITORING : RuntimeFeature("client_feature.android.dropped_frames_reporting")
 }
 
+/**
+ * Known runtime config values.
+ * @param configName the runtime key to use for this config value
+ * @param defaultValue The value
+ */
+sealed class RuntimeConfig(
+    val configName: String,
+    val defaultValue: Int,
+) {
+    /**
+     * The configured value for [jankHeuristicMultiplier] from [JankStatsMonitoring]. More info at https://developer.android.com/topic/performance/jankstats#jank-heuristics
+     */
+    data object JANK_FRAME_HEURISTICS_MULTIPLIER : RuntimeConfig("client_feature.android.jank_frame_heuristics_multiplier", 2)
+
+    /**
+     * The upper bound threshold that defines what constitutes a FROZEN frame reported via [JankStats]
+     *
+     * The default value is 700ms
+     *
+     * Slow Frame: >=16ms to < FROZEN_FRAME_THRESHOLD_IN_MILLI_SECONDS
+     * Frozen Frame: >=FROZEN_FRAME_THRESHOLD_IN_MILLI_SECONDS to <ANR_FRAME_THRESHOLD_IN_MILLI_SECONDS
+     * ANR Frame: >=ANR_FRAME_THRESHOLD_IN_MILLI_SECONDS
+     */
+    data object FROZEN_FRAME_THRESHOLD_IN_MILLI_SECONDS : RuntimeConfig("client_feature.android.frozen_frame.threshold_ms", 700)
```

Comment:
I would just shorten to `FROZEN_FRAME_THRESHOLD_MS` which is common usage of suffixes for units